smf/switch_zone_setup/manifest.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+
+<service_bundle type='manifest' name='switch_zone_setup'>
+
+<service name='system/illumos/switch_zone_setup' type='service' version='1'>
+  <create_default_instance enabled='false' />
+  <single_instance />
```

Comment:
We should omit `<single_instance/>` here.  I know it's common in manifests that ship with the OS, but that's some historical detritus we ought to clean up.  I have also just filed an illumos bug about that: [15477 <single_instance> in service bundles is obsolete](https://www.illumos.org/issues/15477).

Reply:
Done, thanks. I also took the liberty of removing all the `<single_instance/>` from the other manifests.